app/models/project.rb

```diff
@@ -125,6 +126,8 @@ class Project < ActiveRecord::Base
 
   validate :can_clone_repo, if: :validate_repo_connectivity
 
+  validates :article_webhook_url, :allow_blank => true, :format => URI::regexp(%w(http https))
```

Comment:
* use `allow_nil` instead of `allow_blank` and add it to the list of `set_nil_if_blank` fields
* List validators (format) before options (allow_*)